Tests/Receipt/Responses/PrepaidCardReceiptNextPageResponse.json

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
since there's no pagination, is this response still required?

Reply:
Not required. removed all the pagination related objects.